### PR TITLE
add hint to "time2000" parameter in LogItem component

### PR DESCRIPTION
**Pull request by aleks-and**
also

update "jooby-codec" dependency

### CODE REVIEW

File: src/js/components/LogItem/utils/modifyTime2000Parameter.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/* eslint-disable
```
Comment (DarkPark) [inline src/js/components/LogItem/utils/modifyTime2000Parameter.ts:1]: modifyTime2000Properties?

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: src/js/components/LogItem/utils/modifyTime2000Parameter.ts (hunk 1 above)
Comment (aleks-and) [reply src/js/components/LogItem/utils/modifyTime2000Parameter.ts:1]: \+

File: src/js/components/LogItem/utils/formatUTCDate.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ * Formats a JavaScript Date object into a specific UTC format.
+ * The output format is 'YYYY.MM.DD HH:MM:SS GMT'.
+ *
+ * @param date - The Date object to format.
+ * @returns The formatted date string.
+ */
+export default (date: Date): string => {
+    const year = date.getUTCFullYear();
+    // add 1 because months are 0-indexed
+    const month = ('0' + (date.getUTCMonth() + 1).toString()).slice(-2);
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply src/js/components/LogItem/utils/formatUTCDate.ts:11]: applied

File: src/js/components/LogItem/utils/modifyTime2000Parameter.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+/**
+ * Modifies "time2000" properties in a JSON object.
+ *
+ * This function recursively traverses the object and replaces any property
+ * whose name contains "time2000" (case-insensitive) with a date string.
+ *
+ * @param obj - The object to modify.
+ * @return A new object with modified "time2000" properties.
+ */
+const modifyTime2000Parameter = (obj: any): any => {
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply src/js/components/LogItem/utils/modifyTime2000Parameter.ts:24]: applied

File: src/js/components/LogItem/utils/modifyTime2000Parameter.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    if (typeof copy !== 'object' || copy === null) {
+        return copy;
+    }
+
+    if (Array.isArray(copy)) {
+        return copy.map(element => modifyTime2000Parameter(element));
+    }
+
+    const newObj: {[key: string]: any} = {};
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply src/js/components/LogItem/utils/modifyTime2000Parameter.ts:35]: applied

**Review by DarkPark — APPROVED** [review]

**Pull request merged**